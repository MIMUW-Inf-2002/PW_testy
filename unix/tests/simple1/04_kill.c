#include <unistd.h>
#include <stdio.h>

const int period = 100000; // 100 milliseconds

int main() {
    fprintf(stdout, "line 1\n");
    fflush(stdout);

    usleep(period * 10);

    return 0;
}