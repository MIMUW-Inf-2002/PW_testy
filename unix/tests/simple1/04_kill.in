run ./04_kill
run ./04_kill
run ./04_kill
run ./04_kill
run ./04_kill
run ./04_kill
kill 0
kill 1
kill 2
sleep 1200
