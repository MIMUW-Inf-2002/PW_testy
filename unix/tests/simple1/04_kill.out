Task 0 started: pid 57121.
Task 1 started: pid 57125.
Task 2 started: pid 57127.
Task 3 started: pid 57131.
Task 4 started: pid 57137.
Task 5 started: pid 57141.
======
Task 0 ended: signalled.
Task 1 ended: signalled.
Task 2 ended: signalled.
======
======
Task 3 ended: status 0.
Task 4 ended: status 0.
Task 5 ended: status 0.
======
